style.css

```diff
@@ -1,403 +1,1083 @@
+.logo {
+  color: #6071fe;
+  font-size: 1.5em;
+}
+
+body {
+  background: #f7f7f7;
+  font-family: Poppins, sans-serif;
```

Comment:
- [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins), kindly use the right fonts for this project so it looks like the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220825_180735](https://user-images.githubusercontent.com/25227022/186727564-f9e3efd3-ad8b-49b7-bd33-19eee2297783.png)|![Screenshot_20220825_181146](https://user-images.githubusercontent.com/25227022/186728157-5630fc41-ff9d-442b-b5f4-83198183b201.png)|